proto/penumbra/penumbra/core/component/shielded_pool/v1/shielded_pool.proto

```diff
@@ -26,6 +26,14 @@ message GenesisContent {
   repeated Allocation allocations = 3;
 }
 
+message FmdMetaParameters {
+  // Because this used to be FmdParameters, which used as_of_block_height
+  reserved 2;
+  oneof algorithm {
+    uint32 fixed_precision_bits = 1;
+  }
+}
+
```

Comment:
This should just cleanly define the new parameters we want, rather than aiming for compatibility. We don't want to carry forward the old fields forever in new structures, we should just deprecate them in the one place they were used.